test/python/test_roles_api.py

```diff
@@ -727,5 +728,41 @@ def test_delete_member_422(self):
 
         self.assertEqual(context.exception.status, 422)
 
+    def test_get_role_graph_200(self):
+        """Test case for get_role 200 response with graph query param"""
+        details, status, _ = self.api.get_role_with_http_info(
+            self.account,
+            'user',
+            'admin',
+            graph=''
+        )
+
+        self.assertEqual(status, 200)
+        # root policy as parent should always be first for admin user
+        for i in details:
+            self.assertIn('parent', i)
+            self.assertIn('child', i)
+
+    def test_get_role_graph_400(self):
+        """Test case for get_role 400 response with graph query param"""
+        with self.assertRaises(openapi_client.exceptions.ApiException) as context:
+            self.bad_auth_api.get_role(self.account, 'user', '\00', graph='')
```

Comment:
This is stored as a variable
```suggestion
            self.bad_auth_api.get_role(self.account, 'user', 'NULL_BYTE', graph='')
```